learning/books.md

```diff
@@ -1,7 +1,10 @@
 ## Books
 
 @@tight-list
-* Christopher P. Adams. [Learning Microeconometrics with R](https://sites.google.com/view/microeconometricswithr/table-of-contents), January 2021; Has Julia code for all 12 chapters.
+* Mykel J. Kochenderfer, Tim A. Wheeler, and Kyle H. Wray. 
+[Algorithms for Decision Making](https://algorithmsbook.com/), December 2020.
+* Christopher P. Adams. [Learning Microeconometrics with R](https://www.routledge.com/Learning-Microeconometrics-with-R/Adams/p/book/9780367255381), December 2020; 
+Has [Julia code](https://sites.google.com/view/microeconometricswithr/home?authuser=0) for all [12 chapters](https://sites.google.com/view/microeconometricswithr/table-of-contents?authuser=0).
```

Comment:
I think this needs to be indented in order not to mess up the formatting.

Reply:
@StefanKarpinski thanks, I'm not very experienced w/ Github. 
Did my edit to the indenting work?